README.md

```diff
@@ -3,3 +3,9 @@
 [![Build Status](https://travis-ci.org/ivan-podorozhnyi/ssh_tasks.svg?branch=master)](https://travis-ci.org/ivan-podorozhnyi/ssh_tasks)
 
 Simple scripts to interact with remote unix virtual machine
+
+How to run?
+===========
+Automated tests are supposed to run using pytest. For example: 
+
+`pytest <path to your tests>`
```

Comment:
Seems, that here have to be not a general information but how to run this ssh automated tests.
So, i suggest to have here something like:

How to run automated tests?
===========================
Run `pytest ssh_automated_tests` within your virtual environment.